src/library/components/Accordion.tsx

```diff
@@ -47,7 +47,7 @@ const Accordion = ({ headingLevel = 'h2', isContained = false, children }: Props
   };
 
   return (
-    <div className={'bsds-accordion' + (isContained ? ' is-contained' : '')}>
+    <div className={'bsds-accordion' + (isContained ? '-contained' : '')}>
```

Comment:
This will add either bsds-accordion or bsds-accordion-contained to the div. It was originally written to add both, which is what the scss is expecting. So we either need to chain the classes instead of toggling them, or rewrite the scss to handle toggling classes instead of chaining.

Reply:
i lean towards chaining and using the is- prefix, but i think we should look around the codebase to find places we're chaining vs replacing and see what we like best.